examples/example-node-runner/package.json

```diff
@@ -10,11 +10,13 @@
     "lint": "biome check ./src",
     "lint:fix": "biome check --apply ./src",
     "typecheck": "tsc --noEmit",
-    "test": "node --test --loader ts-node/esm src/*.test.ts",
+    "test": "node --test --test-reporter=@voxpelli/node-test-pretty-reporter --loader ts-node/esm src/*.test.ts",
```

Comment:
This uses this other reporter, which takes advantage of the errors that earl generates to print a diff: https://github.com/voxpelli/node-test-pretty-reporter

I think this should be documented.